### ISSUE TITLE
Button 'Add...' cannot be accessed for Screen Reader users

### ISSUE THREAD

**Issue by Deyce1988** [Kyrodan/KeeAnywhere#437]
OS: Current Winn11
Keepass: 2.57
KeeAnywhere: 2.1.0

When using a Screen Reader like NVDA and the Keyboard, adding an account isn't possible, because the Add...-Button in the KeeAnywhere Setting doesn't work when pressing ENTER or SPACE on it.
